ecl2df/hook_implementations/jobs.py

```
@@ -4,8 +4,8 @@
 from pkg_resources import resource_filename
 
 try:
-    from ert_shared.plugins.plugin_manager import hook_implementation
-    from ert_shared.plugins.plugin_response import plugin_response
+    from ert.shared.plugins.plugin_manager import hook_implementation
+    from ert.shared.plugins.plugin_response import plugin_response
 except ModuleNotFoundError:
```

(this will mask errors)

---

ah, yes of course